### PR TITLE
Controller specs

### PR DESCRIPTION
**Pull request by rbd3**
# Change made:
👉 Add RSpec as a gem to my project.
👉 Create a Request spec file for all my controllers.
👉 Add test for status success
👉 Add test for correct template
👉 Add test for body placeholder

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
```
Comment (mukhammadrizooff) [inline spec/requests/posts_spec.rb]: - [ ] [REQUIRED] Good job so far! I appreciate what you have done but `index` and `show` actions for `User` and `Post` controllers should be tested like that 🙏🏼 

```ruby
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
```
Comment (mukhammadrizooff) [inline spec/requests/users_spec.rb]: - [ ] [REQUIRED] Good job so far! I appreciate what you have done but `index` and `show` actions for `User` and `Post` controllers should be tested like that 🙏🏼 

```ruby
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @rbd3, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @rbd3 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ `index` and `show` actions are tested
-  ✅ Implemented suggested changes 
-  ✅ Linters are working properly
-  ✅ All tests are passing

image


## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**